ui/main/src/app/business/services/user.service.ts

```diff
@@ -8,87 +7,117 @@
     private ngUnsubscribe = new Subject<void>();
     private _userRightsPerProcessAndState: Map<string, {rights: RightsEnum, filteringNotificationAllowed: boolean}>;
     private _receiveRightPerProcess: Map<string, number>;
 
     /**
      * @constructor
-     * @param httpClient - Angular build-in
+     * @param userServer - Angular build-in
      */
-    constructor(private httpClient: HttpClient, protected loggerService: OpfabLoggerService, protected alertMessageService: AlertMessageService) {
+    constructor(private userServer: UserServer, 
+        protected loggerService: OpfabLoggerService, protected alertMessageService: AlertMessageService) {
         super(loggerService, alertMessageService);
-        this.userUrl = `${environment.urls.users}`;
-        this.connectionsUrl = `${environment.urls.cards}/connections`;
-        this.willNewSubscriptionDisconnectAnExistingSubscriptionUrl = `${environment.urls.cards}/willNewSubscriptionDisconnectAnExistingSubscription`;
         this._userRightsPerProcessAndState = new Map();
         this._receiveRightPerProcess = new Map();
     }
 
     deleteById(login: string) {
-        const url = `${this.userUrl}/users/${login}`;
-        return this.httpClient.delete(url).pipe(catchError((error: HttpErrorResponse) => this.handleError(error)));
+        return this.userServer.deleteById(login).pipe(catchError((error: HttpErrorResponse) => this.handleError(error)));
     }
 
     getUser(user: string): Observable<User> {
-        return this.httpClient.get<User>(`${this.userUrl}/users/${user}`);
+        return this.userServer.getUser(user).pipe(
+            map((userResponse) => {
+                if (userResponse.status === ServerResponseStatus.OK){
+                    return userResponse.data;
+                } else {
+                    this.handleServerResponseError(userResponse);
+                    return null;
+                }
+            })
+            );
     }
 
     synchronizeWithToken(): Observable<User> {
-        return this.httpClient.post<User>(`${this.userUrl}/users/synchronizeWithToken`, null);
+        return this.userServer.synchronizeWithToken().pipe(
+            map((userResponse) => {
+                if (userResponse.status === ServerResponseStatus.OK){
+                    return userResponse.data;
+                } else {
+                    this.handleServerResponseError(userResponse);
+                    return null;
+                }
+            })
+            );
     }
 
     currentUserWithPerimeters(): Observable<UserWithPerimeters> {
-        return this.httpClient.get<UserWithPerimeters>(`${this.userUrl}/CurrentUserWithPerimeters`);
+        return this.userServer.currentUserWithPerimeters().pipe(
+            map((userResponse) => {
+                if (userResponse.status === ServerResponseStatus.OK){
+                    return userResponse.data;
+                } else {
+                    this.handleServerResponseError(userResponse);
```

Comment:
In this case , do not use handleServerResponse but the logger : 
logger.error("Impossible to load user perimeter")